unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Raycast/Handler/RaycastHandler.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using DCL;
+using DCL.Components;
+using DCL.Controllers;
+using DCL.ECS7;
+using DCL.ECSComponents;
+using DCL.ECSRuntime;
+using DCL.Helpers;
+using DCL.Models;
+using UnityEngine;
+using RaycastHit = DCL.ECSComponents.RaycastHit;
+
+namespace DCLPlugins.ECSComponents.Raycast
+{
+    public class RaycastComponentHandler : IECSComponentHandler<PBRaycast>
+    {
+        private IECSComponentWriter componentWriter;
+        private DataStore_ECS7 dataStore;
+        private LayerMask layerMaskTarget;
+        
+        public RaycastComponentHandler(IECSComponentWriter componentWriter, DataStore_ECS7 dataStore)
```

Comment:
we can remove `DataStore_ECS7 dataStore` from the constructor, it doesn't seem that we are going to need it